examples/tpm_io.c

```diff
@@ -272,6 +286,26 @@
 
             close(spiDev);
         }
+
+    #ifdef WOLFTPM_AUTODETECT
+        /* if response is not 0xFF then we "found" something */
+        if (!foundSpiDev) {
+            if (rxBuf[0] != 0xFF) {
+        #ifdef DEBUG_WOLFTPM
+                printf("Found TPM @ %s\n", TPM2_SPI_DEV);
+        #endif
+                foundSpiDev = 1;
+            }
+            else {
+                int devLen = (int)XSTRLEN(TPM2_SPI_DEV);
+                if (TPM2_SPI_DEV[devLen-1] == '0') {
```

Comment:
A comment for this string manipulation would be nice here: "Change from CS0 to CS1 for next round of detection" or something even simpler :)